test/functional/long/typing-stress-e2e-specs.js

```diff
@@ -7,11 +7,14 @@ import { initSession, deleteSession } from '../helpers/session';
 chai.should();
 chai.use(chaiAsPromised);
 
+// leave the long test to Travis
+const TYPING_TRIES = process.env.TRAVIS ? 200 : 10;
```

Comment:
No. This is specifically to test when you do this a lot. There have been problems with WDA dying in that case.